content/en/modules/HPC/index.md

```diff
@@ -30,7 +30,7 @@ The estimated time to complete this training module is 4h.
 
 The prerequisites to take this module are:
  * the [installation](/modules/installation) module.
- * the [introduction to the terminal](/modules/terminal) module.
+ * the [introduction to the terminal](/modules/introduction_to_terminal) module.
```

Comment:
if possible, can you build the website with the [instruction](https://github.com/school-brainhack/school-brainhack.github.io#run-locally) to verify the URL change is working?

Reply:
Yes, I will! Something did not work out as intended because the link still appears to be broken.

Reply:
I will need to build your changes on my side to check this so I will leave it until later....

Reply:
I tested all the links, and it's working on my side !